src/chttpd/src/chttpd_db.erl

```diff
@@ -372,9 +369,9 @@ create_db_req(#httpd{}=Req, DbName) ->
             throw(Error)
     end.
 
-delete_db_req(#httpd{}=Req, DbName) ->
+delete_db_req(#httpd{user_ctx=Ctx}=Req, DbName) ->
     couch_httpd:verify_is_server_admin(Req),
-    case fabric2_db:delete(DbName, []) of
+    case fabric2_db:delete(DbName, [{user_ctx, Ctx}]) of
```

Comment:
The same applies here as with `fabric2_db:create/2`. We overwrite `#user_ctx` with `?ADMIN_CTX` so it gets tossed away. At least in the create case we could argue that tests could use new `#user_ctx` context on the new Db handle, here it isn't.